pybikes/data/callabike.json

```diff
@@ -3,10 +3,10 @@
         {
             "tag": "callabike-aschaffenburg",
             "meta": {
-                "latitude": 49.436467,
```

Comment:
What I just realized is, that the "id" seems to be a timestamp. So maybe there has to be a code change. I will have look at that.